digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -995,7 +997,10 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
           "track" -> link.attributes.getOrElse("VIITE_TRACK",  link.attributes.get("TEMP_TRACK")),
           "startAddrMValue" -> link.attributes.getOrElse("VIITE_START_ADDR", link.attributes.get("TEMP_START_ADDR")),
           "endAddrMValue" ->  link.attributes.getOrElse("VIITE_END_ADDR", link.attributes.get("TEMP_END_ADDR")),
-          "administrativeClass" -> link.administrativeClass.value
+          "administrativeClass" -> link.administrativeClass.value,
+          "constructionType" -> extractIntValue(link.attributes, "constructionType"),
+          "linkType" -> ( if (extractIntValue(link.attributes, "linkType") != None) LinkType(extractIntValue(link.attributes, "linkType").asInstanceOf[Int]) else LinkType(99) ),
```

Comment:
We should use "UnknownLinkType.value" instead "LinkType(99)"
```scala
"linkType" -> (if (extractIntValue(link.attributes, "linkType") != None) LinkType(extractIntValue(link.attributes, "linkType").asInstanceOf[Int]) else UnknownLinkType.value),

Reply:
Done